mail/dmarc-report-converter/Portfile

```
@@ -0,0 +1,64 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/tierpod/dmarc-report-converter v0.6.5
```

The format for `go.setup` is:

```
go.setup <package url> <version> <version_prefix> <version_suffix>
```

...so this should be:

```suggestion
go.setup            github.com/tierpod/dmarc-report-converter 0.6.5 v
```

(`... 0.6.5 v` instead of `... v0.6.5`)

This is a bug that has been observed in `go2port`.